.github/workflows/backend.yml

```
@@ -102,7 +102,7 @@ jobs:
         run: |
           python -m pip install --upgrade "pip"
           python -m pip install --upgrade "build"
-          python -m pip install --upgrade "scikit-learn ==${{ matrix.scikit-learn }}"
+          python -m pip install --upgrade "scikit-learn~=${{ matrix.scikit-learn }}.0"
```

```suggestion
          python -m pip install --upgrade "scikit-learn ==${{ matrix.scikit-learn }}.*"
```

I prefer to have less changes. Can you add a comment to explain that without this, it will not update to last bugfix?

---

Yep. done